guides/kubernetes/kubernetes.md

```diff
@@ -1,14 +1,14 @@
 # Kubernetes
 
-Kubernetes(also written as K8s) is an open source system for automatic,scaling and management of [containers applications](https://searchitoperations.techtarget.com/definition/application-containerization-app-containerization).It is the most popular tool among all currently present which is used mostly by all the major giants of the industry including Google who actually founded it,Amazon and Microsoft etc.
+Kubernetes(also written as K8s) is an open source system for automatic,scaling and management of [containerized applications](https://searchitoperations.techtarget.com/definition/application-containerization-app-containerization).It is the most popular tool among all currently present which is used mostly by all the major giants of the industry including Google who actually founded it,Amazon and Microsoft etc.
 
-The Github repository of kubernates project can be visited from [here](https://github.com/kubernetes/kubernetes)
+The Github repository of kubernetes project can be visited from [here](https://github.com/kubernetes/kubernetes)
 
 Before knowing it further you must be aware of what actually a docker is,what are containers and containerized applications. 
 
 ## Why kubernetes ?
 
-As i mentioned that kubernetes is a product of Google which was released in July 2015 after a decade and a half years effort of Google who was earlier facing with its own internal Borg system which had certain drawbacks.So they thought of creating something new which can be standardized and can be used by all and the result was kubernetes.
+As i mentioned that kubernetes is a product of Google which was released in July 2015 after a decade and a half year long efforts by the team who were earlier facing difficulties with its own internal Borg system which had certain drawbacks.So they thought of creating something new which can be standardized and can be used by all and the result was kubernetes.
```

Comment:
As I suggested before, please refrain from using the first person. As in, remove "As I mentioned".